Automattic/wp-calypso
# Jetpack Cloud: Use dedicated functions for backups paths ( aka single source of truth )

#### Changes proposed in this Pull Request

* create new `backups/paths.ts` file with the single source of truth for backups paths
* use these new function everywhere

#### Testing instructions

Verify all the following still work:

* Navigating to `/backups` prompts the site picker
* Selecting a site successful navigates to `/backups/:site`
* The "Download backup" and "Restore to this point" buttons on the backups status page
* The "View all backups details" button on the backup status of pages with daily backup
* Clicking "Status" in the sidebar while in the Scan or Settings section
